### DISCUSSION TITLE
Cannot find a descendant at path [1,0] in node: {"children":

### DISCUSSION THREAD

**Discussion by karthikcodes6** [BitPhinix/slate-yjs#325] [Q&A] [answered]
I'm recently getting this error and the whole app crashes. Any idea what would be the root cause?

**Comment by cmmartin** [comment]
I also get this occasionally and it seems to fix itself with a refresh 🤷‍♂️

**Comment by cmmartin** [comment, marked as answer]
@karthikcodes6 In my case, this was caused by setting `autoFocus` on the `Editable` component from Slate. Setting it to false seems to have solved the issue.

I think `autoFocus` causes slate to try to focus the editor before the yjs provider has synced (before it has any nodes to focus). It's basically a race condition

**Reply by BitPhinix** [reply]
Yes, exactly. You can avoid this race condition by conditionally rendering the editable based on `value.length !== 0`

**Reply by BitPhinix** [reply]
I've updated the example with a spinner to show how this is done.

**Comment by AlmazHecker** [comment]
yooo guys, it worked for me:
```
  const editor = useMemo(() => withHistory(withReact(createEditor())), []);

  if (editor.children.length === 0) {
    editor.children.push({
      type: "paragraph",
      children: [{ text: "" }],
    });
  }
```
